@AH-64D Apache Longbow/Addons/fza_ah64_mpd/headers/mfdConstants.h

```diff
@@ -299,6 +299,11 @@
 #define MFD_IND_ACUTIL_COLL  3
 #define MFD_IND_ACUTIL_TRIM  4
 
+#define MFD_IND_ABR_PAGE 0
+#define MFD_IND_ABR_PAGENUM 1
+//abr pages
+#define MFD_ABR_PAGENUM 3
```

Comment:
Maybe a better name rather than "page number" would be "number of pages" - i.e. `NUMPAGES`?